command/config.go

```diff
@@ -0,0 +1,66 @@
+package command
+
+import (
+	"fmt"
+	"github.com/spf13/cobra"
+)
+
+func init() {
+	RootCmd.AddCommand(cfgCmd)
+}
+
+var cfgCmd = &cobra.Command{
+	Use:   "config",
+	Short: "Set and get gh settings",
+	RunE:  cfg,
+	Long: `
+	TODO
+`,
+}
+
+func cfg(cmd *cobra.Command, args []string) error {
+	// TODO decide on top level behavior. I'm thinking either just dump the config or go into an
+	// interactive mode where config keys can be searched through.
+
+	// otherwise, we'll dispatch on arg length for now.
+
+	// TODO i kind of want set to be its own command.
+
+	// TODO just doing this to force config parsing:
+	ctx := contextForCommand(cmd)
+	al, err := ctx.AuthLogin()
+	if err != nil {
+		return err
+	}
+
+	fmt.Println(al)
+
+	switch len(args) {
+	case 0:
+		cfgInteractive(cmd)
```

Comment:
I feel that some sort of interactive mode might be jumping the gun at this point, since we primarily want to allow people to set/change a configuration option.

Reply:
agreed, i was just sketching. i actually meant to take all the command stuff out before even asking you to look at it, sorry